### ISSUE TITLE
Adjustable height for picker bottom sheet

### ISSUE THREAD

**Issue by davemg3** [Daniel-Ioannou/flutter_currency_picker#32]
Hi,

Can you add a property to set the height of bottom sheet or even better to have an adjustable height from few elements until a specific max height (many scrollable elements)

just as for example :  https://pub.dev/packages/map_launcher

thks

**Comment by janar265** [comment]
Hey I created a PR for that in country_picker package. I can add it here aswell.
Also would it make sense to add some constants like 1/3m 2/3 height?

https://github.com/Daniel-Ioannou/flutter_country_picker/pull/73

**Comment by Daniel-Ioannou** [comment]
Released in [v2.0.11](https://pub.dev/packages/currency_picker/versions/2.0.11).
```Dart
showCurrencyPicker(
  context: context,
  theme: CurrencyPickerThemeData(
    bottomSheetHeight: MediaQuery.of(context).size.height / 2,
  ),
  onSelect: (Currency currency) => print('Select currency: ${currency.name}'),
);
```
